gdx/src/com/badlogic/gdx/utils/StreamUtils.java

```diff
@@ -119,9 +119,11 @@ public static String copyStreamToString (InputStream input, int estimatedSize, S
 
 	/** Close and ignore all errors. */
 	public static void closeQuietly (Closeable c) {
-		if (c != null) try {
-			c.close();
-		} catch (Exception ignored) {
+		if (c != null) {
+			try {
+				c.close();
+			} catch (Exception ignored) {
+			}
 		}

```

Comment:
Not sure why this shows up in this PR, I made the change here in 4f8b073610bc1ee955e69d39dd239c2a13de1d85. My personal rule for braces is to use braces if the statement spans multiple lines, for clarity about when the code is executed. I wouldn't normally go around making these kinds of changes, but I had just refactored StreamUtil a bit in 
49a7c11997932e5d1d7974fde292decb566b05b0 and shortly after the commit I noticed `closeQuietly`.